mousestyles/path_diversity/smooth_noise.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import print_function, absolute_import, division
+
+import pandas as pd
+import numpy as np
+from path_features import compute_angles
+from detect_noise import detect_noise
```

Comment:
import statements should be in the following format:
`from directory import file_name`
and when you need to call the function, do `file_name.function_name`.
So in this case, it would be 
`from mousestyles.path_diversity import path_features`
and 
`from mousestyles.detect_noise import detect_noise`